docs/upgrade/focal_migration.rst

```diff
@@ -0,0 +1,309 @@
+
+Instances that have not yet enabled v3 
+:ref:`onion services <glossary_onion_service>` should choose the 
+:ref:`Alternate Migration Procedure <migration_alternate>`. 
+
+
+.. _prune_submissions:
+
+Delete Old Submissions from the Server
+--------------------------------------
+
+In coordination with journalists, ensure that any old or unneeded
+submissions have been deleted from the server. Pruning old submissions
+will reduce the size and improve the speed of your server backup. 
+Journalists can delete unneeded submissions via the *Journalist Interface.*
+
+
+.. _download_focal:
+
+Download and Verify Ubuntu 20.04 LTS (Focal) Installation Media
+----------------------------------------------------------------
+
+Follow our instructions to 
+:doc:`download and verify Ubuntu Server 20.04 LTS <../servers>` and
+install the .iso file onto a USB stick.
+
+You have now completed all the preparatory steps. The rest of the 
+migration procedure will be completed during your maintenance window. 
+
+Migration
+~~~~~~~~~ 
+
+.. _migration_standard:
+
+Standard Migration Procedure
+----------------------------
+Perform these steps if your instance is already using v3 onion services. Ensure
+you have completed the :ref:`preparatory steps <focal_prep>`.
+
+#. Announce your maintenance window. As part of this procedure, your servers
+   will become unreachable.
+#. :doc:`Take a backup of the current instance <../backup_and_restore>`. Once 
+   you have taken a backup of the servers, power them off.
+
+    .. warning::
+
+     The next steps will overwrite existing data on the servers.
+
+     Data from the *Monitor Server* will not be restored after the backup.
+     If you require historical data from the *Monitor Server*, archive it
+     separately before proceeding.
+#. Follow the instructions on 
+   :ref:`hardware migration for instances using v2+v3 or v3 onion services <migrate_v3>`. 
+
+   You will be instructed to install Ubuntu 20.04, overwriting the 
+   existing operating system on the servers,
+   and you will be guided through reinstalling SecureDrop.
+
+   Once you have reinstalled SecureDrop, you will be instructed on how to
+   apply your backup file, restoring data and Tor credentials.  
+#. Ensure that *Journalist* and *Admin Workstations* are up-to-date and can
+   access the *Source* and *Journalist Interfaces*. When your instance 
+   enabled v3 onion services, one of the important steps was to ensure that
+   all Journalist and Admin Workstations were 
+   :ref:`updated with the new Tor credentials <update_tails_v3>`. If you 
+   have not yet completed this step, you will need to do so now.
+
+   .. note::
+
+      If you cannot update your Journalists' Tails USBs in person due
+      to remote work policies, 
+      contact Support for suggestions on how to safely complete this step.
+
+#. Ensure that your Landing Page shows your v3 onion URL. If it does not,
+   :ref:`publish your new Source Interface URL <publish_v3>`.
```

Comment:
Since they already have a v3 URL which we're restoring, it may be less confusing to put this step early in the process, (perhaps first), noting again that any v2 onion services will be disabled as part of this migration.

Reply:
Moved to step 1